.evergreen/config/functions.yml

```diff
@@ -274,12 +274,15 @@ functions:
           export KMS_TLS_CA_FILE="${client_side_encryption_kms_tls_ca_file}"
           export KMS_TLS_CERTIFICATE_KEY_FILE="${client_side_encryption_kms_tls_certificate_key_file}"
           export MONGODB_IS_SERVERLESS=on
-          export MONGODB_USERNAME=${SERVERLESS_ATLAS_USER}
-          export MONGODB_PASSWORD=${SERVERLESS_ATLAS_PASSWORD}
+          export MONGODB_URI="${SERVERLESS_URI}"
           export PATH="${PHP_PATH}/bin:$PATH"
 
+          . ${DRIVERS_TOOLS}/.evergreen/serverless/secrets-export.sh
```

Comment:
This file is created by setup-secrets and contains the secrets needed to run tests